packages/discord.js/src/util/Sweepers.js

```diff
@@ -425,6 +425,9 @@ class Sweepers {
     let items = 0;
 
     for (const guild of this.client.guilds.cache.values()) {
+      // We may be unable to sweep the cache if the guild is unavailable
+      if (!guild.available) continue;
```

Comment:
I know I proposed this as one of two possible fixes. But after thinking about it it seems that the fix to have a fallback in the next line if `guild[key]` was undefined would be the better alternative, because we‘d still want to sweep caches of unavailable guilds if the cache exists. So `if (!(key in guild))` seems to be the better option imho.

Reply:
Makes sense! Pushed a fix